### ISSUE TITLE
Help statement

### ISSUE THREAD

**Issue by bgotink** [kevin-smets/caml#3]
Error output is not really helpful:

```
$ caml
Flag '-i' or '--input' is mandatory. e.g. --input a b c, cascades a.yml, b.yml and c.yml.
```

The `-i` flag doesn't exist anymore, should be `-f` or `--files`

**Comment by kevin-smets** [comment]
Fixed in #8be191b914f6559425a88bd68b9713efd88de187

**Comment by kevin-smets** [comment]
Fixed in 955241800f28e01cce668ff837feb534b2eb04fa
